docs/topics/images.rst

```diff
@@ -295,6 +296,16 @@ For example, to make the tag always convert the image to a JPEG, use ``format-jp
 
 You may also use ``format-png`` or ``format-gif``.
 
+WebP lossless
```

Comment:
Lossless WebP

You can encode the image losslessly into WebP format by using the ``format-webp-lossless`` filter: